exporter/otlphttpexporter/otlp_test.go

```diff
@@ -128,12 +128,12 @@ func TestErrorResponses(t *testing.T) {
 			isPermErr:      true,
 		},
 		{
-			name:           "419",
```

Comment:
Can we keep the test case on 419 and add 429 as a new case?

Reply:
I think this was a typo in the previous test, there is no [http status code 419](https://en.wikipedia.org/wiki/List_of_HTTP_status_codes#4xx_client_errors)